src/app.ts

```diff
@@ -13,14 +13,44 @@ interface TemperatureSummary {
   average: number
 }
 
+class Readings {
+  static readings: TemperatureReading[]
+}
+
 export function processReadings(readings: TemperatureReading[]): void {
   // add here your code
+  Readings.readings = readings.slice()
```

Comment:
do you really need a class? I think an array defined outside this function is good enough

Reply:
also, why are you using slice?